### PR TITLE
workbench | tx-generator: updated reporting and tracing detail

### PR DESCRIPTION
**Pull request by mgmeier**
# Description

This PR
* updates reporting template(s)
* restores classifying the workload type in a downward-compatible manner
* changes `tx-generator` shutdown logic, which could sometimes result in an unjustified exit code 1 due to timing
* restores detail level of `tx-generator` trace messages

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated.  These may include:
  - golden tests
  - property tests
  - roundtrip tests
  - integration tests
  See [Runnings tests](https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests) for more details
- [ ] Any changes are noted in the `CHANGELOG.md` for affected package
- [ ] The version bounds in `.cabal` files are updated
- [X] CI passes. See note on CI.  The following CI checks are required:
  - [X] Code is linted with `hlint`.  See `.github/workflows/check-hlint.yml` to get the `hlint` version
  - [X] Code is formatted with `stylish-haskell`.  See `.github/workflows/stylish-haskell.yml` to get the `stylish-haskell` version
  - [X] Code builds on Linux, MacOS and Windows for `ghc-8.10.7` and `ghc-9.2.7`
- [x] Self-reviewed the diff

### CODE REVIEW

File: bench/tx-generator/src/Cardano/Benchmarking/Script.hs
Comment (NadiaYvette) [inline bench/tx-generator/src/Cardano/Benchmarking/Script.hs:1]: This seems like a future TODO for me to work out why these delays are needed as things are & to eventually move them over to some sort of more orderly shutdown process. Reducing them is definitely good.

File: bench/tx-generator/src/Cardano/Benchmarking/Tracer.hs
Comment (NadiaYvette) [inline bench/tx-generator/src/Cardano/Benchmarking/Tracer.hs:1]: Definitely a good catch that the strings for creation & detail level setting weren't matching.

File: bench/locli/src/Cardano/Analysis/API/Context.hs
Comment (NadiaYvette) [inline bench/locli/src/Cardano/Analysis/API/Context.hs:1]: I like Baldur's suggestion for plutusLoopScript. It would have been a quicker read for me.

**Review by NadiaYvette — APPROVED** [review]
The code & the logs both look good! Sorry it took a long-running re-run to do my own testrun of it all.

**Review by angerman — APPROVED** [review]
Looks generally ok to me. I'm not sure if you want me to look at something specific @mgmeier? I don't see CI/nix/... related changes?

**Pull request merged**